Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -20,6 +20,15 @@ export default function widgetBehavior(publicAPI, model) {
   let isDragging = null;
   let isScrolling = false;
 
+  // Update some widget state attributes when no actors are selected
+  // Usefull to update 'updateMethodeName' to the correct name which
+  // will be TranslateCenter by default
+  publicAPI.updateState = () => {
```

Comment:
I think the function name is misleading. It carries a message that it is an update(), therefore it is safe to call it anywhere.

However, it does not take into account the mouse position, so it can't be called anywhere. It "resets" the updateMethodName state.

Therefore I suggest instead: 
`updateState` -> `resetState` or `resetUpdateMethod`.